### ISSUE TITLE
Follow Up - Catch all transactions outputs from Queries task

### ISSUE THREAD

**Issue by Ben8t** [kestra-io/plugin-jdbc#397]
### Feature description

As a follow up of https://github.com/kestra-io/plugin-jdbc/issues/368

Catch all possible outputs from transactions like INSERT, UPDATE, DELETE and CREATE.
Not every database provide the same values - so it could be a case by case implementation.

**Comment by MoMoUsama** [comment]
Hi @Ben8t,
by "Catch all possible outputs" you mean the no. of affected rows and the generated Keys (if the DB supports keys generation), isn't it ?

**Comment by Ben8t** [comment]
Yes I think this need exploration but the goal is to be able to catch change in the DB so we can use them as outputs of the task and downstream tasks in a Flow
